src/main/java/com/spotify/dns/ServiceResolvingChangeNotifier.java

```diff
@@ -62,16 +60,16 @@
    * @param resolver            The resolver to use.
    * @param fqdn                The name to lookup SRV records for
    * @param resultTransformer   The transform function
-   * @param errorHandler        The error handler that will receive exceptions
+   * @param errorHandler        The error handler that will receive exceptions (nullable)
```

Comment:
Did you need to remove the JSR dependency with `@Nullable`?

Reply:
Yes, based on this post https://blog.codefx.org/java/jsr-305-java-9/